packages/server/graphql/types/SuggestedAction.ts

```diff
@@ -44,7 +44,7 @@ export const suggestedActionInterfaceFields = () => ({
   }
 })
 
-const SuggestedAction = new GraphQLInterfaceType({
+const SuggestedAction: GraphQLInterfaceType = new GraphQLInterfaceType({
```

Comment:
+1 you can fix it by fixing the `resolveTypeLookup` type as it still complains there